code/controllers/subsystem/processing/quirks.dm

```diff
@@ -65,3 +65,74 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 	// Assign wayfinding pinpointer granting quirk if they're new
 	if(cli.get_exp_living(TRUE) < EXP_ASSIGN_WAYFINDER && !user.has_quirk(/datum/quirk/needswayfinder))
 		user.add_quirk(/datum/quirk/needswayfinder, TRUE)
+
+/datum/controller/subsystem/processing/quirks/proc/randomise_quirks(mob/living/user, bonus_points = 0)
+	var/bonus_quirks = max((length(user.roundstart_quirks) + rand(-3,3)), 3)
```

Comment:
What are these 3's from? Can you extract it to a define?

Reply:
Done the define stuff too!